src/onegov/translator_directory/utils.py

```diff
@@ -201,3 +205,13 @@ def geocode_translator_addresses(
             coords_not_found.append(trs)
 
     return trs_total, total, geocoded, skipped, coords_not_found
+
+
+def nationality_choices(locale: str | None) -> list['_Choice']:
+    assert locale
+    _locale = Locale.parse(locale)
+    nationalities = [_locale.territories.get(code) for code in
+                     _locale.territories if len(code) == 2]
+    nationalities = [(v, v) for v in sorted(nationalities)]
```

Comment:
Is there a good reason not to store the code as the value? We already have to format the value for display anyways. Storing the code is more robust, since it would allow for mixed language scenarios in the future.

```suggestion
    nationalities = sorted((
        (code, name) for code, name in
        _locale.territories.items() if len(code) == 2
    ), key=itemgetter(1))
``

You may also want to consider pinning the most likely choices to the top (`CH` and its neighbors), so you don't need to use the search for them.

Reply:
Agree, for multi language support this makes absolutely sense.